src/txmempool.cpp

```diff
@@ -314,11 +314,16 @@ void CTxMemPoolEntry::UpdateDescendantState(int64_t modifySize, CAmount modifyFe
 
 void CTxMemPoolEntry::UpdateAncestorState(int64_t modifySize, CAmount modifyFee, int64_t modifyCount, int64_t modifySigOps)
 {
+    // An unsigned integer wraparound may take place here (modifySize is allowed to be negative).
     nSizeWithAncestors += modifySize;
     assert(int64_t(nSizeWithAncestors) > 0);
```

Comment:
I don't think this can wrap around in normal operation, since it is always first increased by some `modifySize` and only later reduced by the same `modifySize`.

Also, note the assert, which would be hit if a wraparound occurs.